packages/shared/src/components/utilities/loaders/generic.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { ReactElement } from 'react';
+import classnames from 'classnames';
+import { LoaderIcon } from '../../icons';
+import { IconSize } from '../../Icon';
+
+interface Props {
+  className?: string;
+  label?: string;
+}
+
+export const GenericLoader = ({
+  className,
+  label = 'Preparing...',
+}: Props): ReactElement => {
+  return (
+    <div className="fixed inset-0 z-modal flex items-center justify-center bg-background-default">
+      <div className="flex flex-col items-center gap-5">
+        <LoaderIcon
+          size={IconSize.XLarge}
+          className={classnames(
+            className,
+            'flex-shrink-0 animate-spin drop-shadow-[0_0_5px_#CE3DE3]',
```

Comment:
This arbitrary value doesn't seem right.
It's not in our design library or what's the story here?

Reply:
We have the color defined, it's mostly about convincing tailwind that it wants to use the CSS variable and not show it as transparent :/

Reply:
Added a CSS variable for the shadow color and used that instead. Let me know what you think.